vllm/attention/ops/triton_unified_attention.py

```diff
@@ -225,29 +241,29 @@ def kernel_unified_attention_2d(
             S += alibi_slope[:, None] * (seq_offset - context_len)
 
         # compute running maximum
-        # m_j : (BLOCK_M,)
+        # m_j : (BLOCK_Q * num_queries_per_kv,)
```

Comment:
This PR seems to undo the changes from https://github.com/vllm-project/vllm/pull/18100